.github/ISSUE_TEMPLATE/2_bug.md

```diff
@@ -16,36 +16,75 @@ assignees: ''
       * https://api.flutter.dev/
       * https://stackoverflow.com/questions/tagged/flutter?sort=frequent
 
+     Issues directly asking for support will be closed.
+
      If you have found a bug or if our documentation doesn't have an answer
      to what you're looking for, then fill out the template below. Please read
      our guide to filing a bug first: https://flutter.dev/docs/resources/bug-reports
 -->
 
-## Steps to Reproduce
+### Description
+
+<!--
+     Please describe in detail the problem you are running into.
+
+     List concrete steps to reproduce the problem, if this will better illustrate it:
+
+        1. ...
+        2. ...
+        3. ...
+-->
+
+### Expected results
+
+<!-- What did you want to see? -->
 
-<!-- You must include full steps to reproduce so that we can reproduce the problem. -->
+### Actual results
+
+<!-- What did you see? -->
 
-1. Run `flutter create bug`.
-2. Update the files as follows: ... <!-- include every file that is different from the template app! -->
-3. ... <!-- describe how to reproduce the problem -->
```

Comment:
we seem to have lost the `flutter create ...` step here

Reply:
such meta-form of reproduction steps doesn't bring anything productive to triging, it makes people think they should verbosily list every step they made to ***create a reproduction*** rather than describe actual steps to reproduce

i moved the requirement of the reproducible code sample below in the `<details>`

if people come to this issue tracker they definitely know how the create a flutter app, so i excluded this note as not needed

Reply:
`flutter create ...` is necessary so the reporter is able to reproduce the issue outside their production app which would make the issue more likely related to Flutter than third-party packages, issues with their code and other reasons. We've many issues reported where authors follow these steps and provide reproducible from scratch `flutter create..`, modifying `main.dart`,  making changes to `pubspec.yaml` to running the app, which leaves the production app or existing app out as cause the of the issue.

Reply:
if we want to keep concrete instructions how do you create a minimal reproducible, we would like to rephrase this to express them as intructions for submitter, not something he will likely leave as an actual reproductrion step

i.e. that should be something that goes after this paragraph https://github.com/flutter/flutter/pull/79594/files#diff-a5bf860cf261d12c428dcd1164243e6a454f770d07b2b9b4e933d6f76d3051f9R50-R51